tests/integration/test_render.py

```diff
@@ -57,6 +61,10 @@ def _get_style(self, selector, style, computed=True):
             query = 'return $("{selector}").get(0).style.{style}'
         return self.browser.execute_script(query.format(selector=selector, style=style))
 
+    def _get_border_style(self, elt, style):
+        query = 'return window.getComputedStyle(document.getElementById("{elt}"), null).getPropertyValue("{style}")'
+        return self.browser.execute_script(query.format(elt=elt, style=style))
+
```

Comment:
@agaylard I think you can do without this method -- `_get_style` above should do what you want, you just need to change the inputs that you are passing in `test_borders_inactive` and `test_borders_active` a little bit (I'll give more detail in another comment).